lib/config/notification.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+// Default configuration
+const brokers = [process.env?.NOTIFICATION_BROKER] || null;
+
+module.exports = {
+    brokers,
```

Comment:
I believe the notification service from web-ui expects an array;
Could you confirm that the process.env will return an `Array` and not a `String`?

Reply:
Nope, it will not, but that's why `[` and `]` were added.

Reply:
Ah, ok. I missed the brackets as I was thinking we are expecting multiple brokers separated by commas